functional_tui_test.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+set -e
+
+echo "Cleaning up potentially lingering sessions"
```

Comment:
This is not how this should be done. See this [tutorial](https://www.linuxjournal.com/content/use-bash-trap-statement-cleanup-temporary-files)